### ISSUE TITLE
Add support for es6 import

### ISSUE THREAD

**Issue by imaksp** [Yankovsky/nouislider-angular#23]
Please add support for es6 import.
e.g. for ui-router
 `import uiRouter from 'angular-ui-router'`

**Comment by Yankovsky** [comment]
@aks0020 I am a bit short of time, maybe you want to make pr?

**Comment by ekarudianto** [comment]
+1

**Comment by ekarudianto** [comment]
Hi @Yankovsky I've forked your repo and made my self the directive and the support of using es6 here

https://github.com/ekarudianto/nouislider-angular-es6

I put your name as the original creator, would you mind if I publish it to npm ? since I'd like to use it personaly for my current project ?

**Comment by Yankovsky** [comment]
@ekarudianto @aks0020 just to clarify, what is the point of using es6 imports in this lib? It only depends on noUiSlider and assumes that it is available in window object.

**Comment by ekarudianto** [comment]
Yes I was using webpack as the build tool, and it won't work if I just do 'import' from your repo since you're not treat it as a module .. @Yankovsky

**Comment by Yankovsky** [comment]
@ekarudianto @aks0020 do you still need this feature?

**Comment by ekarudianto** [comment]
Hi @Yankovsky, not really anymore no :) sorry for the late reply
